src/nuspec/SandcastleRefMapper/SandcastleRefMapper.nuspec

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/10/nuspec.xsd">
+  <metadata>
+    <id>SandcastleRefMapper</id>
+    <version>1.0.0</version>
+    <title>SandcastleRefMapper</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <licenseUrl>https://github.com/dotnet/docfx/blob/master/LICENSE</licenseUrl>
+    <projectUrl>https://github.com/dotnet/docfx</projectUrl>
+    <description>This is a DocFX tool to convert Sandcastle documentation to refmaps.</description>
```

Comment:
Some minor adjustment: "This is a DocFX tool to generate cross reference map for Sandcastle documentation site.